spec/draft_spec.rb

```diff
@@ -57,29 +57,41 @@
       expect(result.snippet).to_not be ""
     end
 
-    it "when an error occurs, it sets server_error if defined" do
-      stub_request(:post, "https://api.nylas.com/send").with(basic_auth: [@access_token]).
-         to_return(:status => 400,
-                   :body => '{ "message": "Invalid recipient address benbitdiddle@gmailcom", ' +
-                            '  "type": "invalid_request_error", "server_error": "SPAM" }',
-                   :headers => {"Content-Type" => "application/json"})
-
-      draft = Inbox::Draft.new(@inbox)
-      draft.subject = 'Test draft'
-      draft.account_id = @account_id
-      draft.to = [{:name => 'Helena Handbasket', :email => 'helena@nylas.com'}]
-      expect(draft.id).to be nil
-
-      expect { draft.send! }.to raise_error(Inbox::InvalidRequest)
-      begin
-        draft.send!
-      rescue Inbox::InvalidRequest => e
-        expect(e.message).to eq("Invalid recipient address benbitdiddle@gmailcom")
-        expect(e.server_error).to eq("SPAM")
+    error_codes = [[400, Inbox::InvalidRequest],
```

Comment:
Hmm, I would just create an array from `HTTP_CODE_TO_EXCEPTIONS`. No need to duplicate the array here.